HW_8/view.py

```diff
@@ -0,0 +1,10 @@
+def display_menu():
+    print("--------------------------------------")
+    print(" Добро пожаловать в базу данных студентов GeekBrains!")
+    print("---------------------------------------")
+    print("1. Добавить новую запись.")
+    print("2. Показать все записи.")
+    print("3. Найти студента в базе.")
+    print("4. Обновить информацию о студенте.")
+    print("5. Удалить запись.")
+    print("6. Выйти из программы.")
```

Comment:
Можно обойтись и одним принтом.
```suggestion
    print("--------------------------------------\n"
          " Добро пожаловать в базу данных студентов GeekBrains!\n"
          "---------------------------------------\n"
          "1. Добавить новую запись.\n"
          "2. Показать все записи.\n"
          "3. Найти студента в базе.\n"
          "4. Обновить информацию о студенте.\n"
          "5. Удалить запись.\n"
          "6. Выйти из программы.")
```